Cargo.toml

```diff
@@ -3,7 +3,7 @@ name = "rav1e"
 version = "0.6.0"
 authors = ["Thomas Daede <tdaede@xiph.org>"]
 edition = "2021"
-rust-version = "1.60.0"
+rust-version = "1.61.0"
```

Comment:
The `nalgebra` library which `yuvxyb` uses for matrix multiplication to convert to RGB requires rustc 1.61.